src/main/java/eu/mihosoft/vrl/v3d/CSG.java

```diff
@@ -1179,5 +1462,29 @@ public void setOptType(OptType optType) {
         POLYGON_BOUND,
         NONE
     }
+    /**

```

Comment:
Cool! Who is "we" btw? I have always wondered how to become more active in JCSG core development and would like a channel to discuss changes before shooting our broken PRs ;)